src/Sylius/Bundle/AddressingBundle/Form/Type/ZoneChoiceType.php

```diff
@@ -22,15 +22,11 @@
 
 final class ZoneChoiceType extends AbstractType
 {
-    public function __construct(private RepositoryInterface $zoneRepository, private array $scopeTypes = [])
-    {
-        if (count($scopeTypes) === 0) {
-            trigger_deprecation(
```

Comment:
This one should be converted to an exception now to require passing scope types.